server/elasticsearch/src/main/resources/hawkular-apm-mapping.json

```diff
@@ -17,11 +17,6 @@
             }],
             "date_detection" : false,
             "numeric_detection": false,
-            "_timestamp": {
```

Comment:
In our time ranged queries we use this code https://github.com/hawkular/hawkular-apm/blob/master/server/elasticsearch/src/main/java/org/hawkular/apm/server/elasticsearch/ElasticsearchUtil.java#L53

Here is the doc for it: 
https://www.elastic.co/guide/en/elasticsearch/reference/1.7/mapping-timestamp-field.html
I'm not if it would work, since we store it as microseconds.